pkg/apis/abac/v0/doc.go

```diff
@@ -15,6 +15,6 @@ limitations under the License.
 */
 
 // +k8s:deepcopy-gen=package
-
```

Comment:
for a minimal change, plz revert this line. or do the same for other blank lines between `// +..` too.